src/eko/couplings.py

```diff
@@ -610,9 +611,12 @@ def a(
         is_downward = thresholds.is_downward_path(path)
         shift = thresholds.flavor_shift(is_downward)
 
-        # as a default assume mu_F^2 = mu_R^2
-        if fact_scale is None:
-            fact_scale = scale_to
+        # TODO remove in 0.13
+        if fact_scale is not None:
+            warnings.warn(
+                "The `fact_scale` argument is deprecated - use the ThresholdAtlas instead!",
+                DeprecationWarning,
+            )
```

Comment:
As @andreab1997 correctly said, I'd not drop it at any time, just stop using it in our own code.

Reply:
I believe, we don't need this and,in fact, it was wrong before. It was needed before, because we (hard) divided masses on which later we had no trace, so we needed this argument to recover. Instead the correct way is (I believe) to _not_ change the masses, but to use the thresholds - which are effectively doing the same thing, except that we know of it. Note that this was only used for the matching, so I'm not 100% sure this was the "resummation scale" business. In any case the consistent way is to move the ThresholdAtlas (as the message says) if you want to shift the underlaying scale - and only then you're consistent, because you have the correct matching in game.